src/staticanalysis/bot/static_analysis_bot/workflows/remote.py

```diff
@@ -0,0 +1,132 @@
+
+        # Process only the failed tasks
+        # A completed task here means the analyzer did not find any issues
+        if state == 'completed':
+            logger.info('No issues detected by completed task', id=self.task_id)
+            return
+        elif state != 'failed':
+            logger.warn('Unsupported task state', state=state, id=self.task_id)
+            return
+
+        # Load artifact logs from the last run
+        self.run_id = status['status']['runs'][-1]['runId']
+        artifacts = queue_service.listArtifacts(self.task_id, self.run_id)
+        assert 'artifacts' in artifacts, 'Missing artifacts'
+        logs = [
+            artifact['name']
+            for artifact in artifacts['artifacts']
+            if artifact['storageType'] != 'reference' and artifact['contentType'].startswith('text/')
+        ]
+
+        # Load logs from artifact API
+        out = {}
+        for log in logs:
+            logger.info('Reading log', task_id=self.task_id, log=log)
+            try:
+                artifact = queue_service.getArtifact(self.task_id, self.run_id, log)
+                assert 'response' in artifact, 'Failed loading artifact'
+                out[log] = artifact['response'].content
+            except Exception as e:
+                logger.warn('Failed to read log', task_id=self.task_id, run_id=self.run_id, log=log, error=e)
+        return out
+
+
+class RemoteWorkflow(object):
+    '''
+    Secondary workflow to analyze the output from a try task group
+    '''
+    def __init__(self, queue_service):
+        # Use TC services client
+        self.queue_service = queue_service
+
+    def run(self, revision):
+        assert settings.taskcluster.task_id is not None, 'Missing Taskcluster task id'
+
+        # Load task description
+        task = self.queue_service.task(settings.taskcluster.task_id)
+        assert len(task['dependencies']) > 0, 'No task dependencies to analyze'
+
+        # Find issues in dependencies
+        issues = []
+        for dep_id in task['dependencies']:
+            try:
+                task = AnalysisTask(dep_id)
+                logs = task.load_logs(self.queue_service)
+                if logs is None:
+                    continue
+
+                for log in logs.values():
+                    issues += self.parse_issues(task, log, revision)
+            except Exception as e:
+                logger.warn('Failure during task analysis', task=settings.taskcluster.task_id, error=e)
+                continue
+
+        return issues
+
+    def parse_issues(self, task, log_content, revision):
+        '''
+        Parse issues from a log file content
+        '''
+        assert isinstance(task, AnalysisTask)
+
+        # Lookup issues using marker
+        issues = [
+            line[line.index(ISSUE_MARKER) + len(ISSUE_MARKER):]
+            for line in log_content.decode('utf-8').split('\r\n')
+            if ISSUE_MARKER in line
```

Comment:
Don't failure lines always *start* with ISSUE_MARKER?

Reply:
Nope, i get the full log from Taskcluster (so cloning, updating to revision, etc...)

Reply:
No I mean, couldn't this be `if line.startswith(ISSUE_MARKER)` instead of `if ISSUE_MARKER in line` (and two lines before just `line[len(ISSUE_MARKER):]` instead of `line[line.index(ISSUE_MARKER) + len(ISSUE_MARKER):]`)?

Reply:
Nope, we can get (or not!) the log times from mach. I saw several different outputs.

I think this code will be removed soon enough, when i'll work on before/after in those tasks and outputs json artifacts